drake/examples/Cars/simulateLCM.cpp

```diff
@@ -78,11 +89,22 @@ int main(int argc, char* argv[]) {
   // be reused
   DrakeJoint::FloatingBaseType floating_base_type = DrakeJoint::QUATERNION;
 
-
   auto rigid_body_sys = make_shared<RigidBodySystem>();
-  rigid_body_sys->addRobotFromFile(argv[1], floating_base_type);
-  auto const & tree = rigid_body_sys->getRigidBodyTree();
-  for (int i=2; i<argc; i++)
+
+  // The following variable, weld_to_frame, is only needed if the model is a
+  // URDF file. It is needed since URDF does not specify the location and
+  // orientation of the car's root node in the world. If the model is an SDF,
+  // weld_to_frame is ignored by the parser.
```

Comment:
> @param model_to_world_transformation The transformation between this RigidBodySystem's world coordinate frame to the model's world coordinate frame.

The name and description of the transform are ambiguous. I am guessing that what you intend is to give the pose of the model frame in the world. If so, how about: `@param pose_of_model_in_world Transform giving the pose of the file's model frame expressed in the world frame.` I would go further and establish a firm discipline throughout Drake about the way we think about frames and the language and notation we use, but that's too much for this PR to deal with! (Would like to discuss while I'm out there.)